src/params.ts

```diff
@@ -12,7 +12,7 @@ export const anyEntity = '0xffffffffffffffffffffffffffffffffffffffff'
 // - Local testnet node to connect to (testnetPort)
 // - ENS address to resolve names (aragenMnemonic)
 // In order to deploy the Aragon bases a specific gas limit is needed (aragenGasLimit).
-export const testnetPort = 8545
+export const testnetPort = 48545
```

Comment:
I think we should probably still keep the default port at 8545, since most software is set up to default to this (e.g. a custom network in metamask, etc).

Reply:
Yes, I agree. I changed this constant because otherwise tests would fail, but we can simply define different port number for tests and use 8545 as default for buidling.

Reply:
Done.